images/os/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM fedora:29 AS build
```

Comment:
could we do latest or at least bump up to 31 or 32?

Reply:
All the openshift tests use f29 for injecting the rpm into the machine-os-content, so going to leave it at 29 for consistence.